Safety/Inc/safetyConfig.hpp

```diff
@@ -1,14 +1,19 @@
 #ifndef SAFETY_CONFIG_HPP
 #define SAFETY_CONFIG_HPP
 
-// pwm order: left tail, right tail, aileron, nose wheel, throttle, gimbal x, gimbal y, reverse thrust.
-constexpr static uint8_t PWM_CHANNEL_LEFT_TAIL {0};
-constexpr static uint8_t PWM_CHANNEL_RIGHT_TAIL {1};
-constexpr static uint8_t PWM_CHANNEL_AILERON {2};
-constexpr static uint8_t PWM_CHANNEL_NOSE_WHEEL {3};
-constexpr static uint8_t PWM_CHANNEL_THROTTLE {4};
-constexpr static uint8_t PWM_CHANNEL_GIMBAL_YAW {5};
-constexpr static uint8_t PWM_CHANNEL_GRABBER {6};
+// ppm channels so we can set and forget here.
+// TODO: Add channels for Arm/Disarm (later also grabber controls)
+constexpr uint8_t LONGITUDE_PPM_CHANNEL {0};
+constexpr uint8_t LATITUDE_PPM_CHANNEL {1};
+constexpr uint8_t VERTICAL_PPM_CHANNEL {2};
```

Comment:
Do we want to use VERTICAL or ALTITUDE.

Up to you, just as long as we are consistent.

Reply:
honestly looking back at this throttle might be more effective a name. Also, longitude and latitude should prob be changed to pitch and roll respectively

Reply:
Don't know about throttle (since throttle can refer to % speed in any direction of motion).....